web/styles/left_sidebar.css

```diff
@@ -540,6 +540,72 @@ li.topic-list-item:hover .topic-sidebar-menu-icon {
     color: hsl(0deg 0% 53%);
 }
 
+.top-left-sidebar-menu-icon {
+    display: none;
+}
+
+.icon_layout {
+    padding: 10px 0 5px;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+
+    .top_left_texts {
+        display: none;
+    }
+
+    .filter-icon i {
+        padding-right: 0 !important;
+    }
+
+    .unread_count {
+        position: absolute;
+        bottom: 1rem;
+        left: 1.2rem;
+    }
+
+    .top_left_starred_messages .unread_count,
+    .top_left_drafts .unread_count {
+        background-color: hsl(0, 0.55%, 71.18%) !important;
+        border: 0 !important;
+        color: hsl(0, 0%, 100%) !important;
+        opacity: 0.8;
+    }
+
+    .fa,
+    i.fa-at,
+    i.fa-star {
+        font-size: 20px;
+    }
+
+    .all-messages-sidebar-menu-icon,
+    .starred-messages-sidebar-menu-icon,
+    .drafts-sidebar-menu-icon {
+        display: none;
+    }
+
+    .top_left_all_messages:hover .all-messages-sidebar-menu-icon,
+    .top_left_starred_messages:hover .starred-messages-sidebar-menu-icon,
+    .top_left_drafts:hover .drafts-sidebar-menu-icon {
+        display: none;
+    }
+
+    .top-left-sidebar-menu-icon {
+        height: 15px;
+        width: 15px;
+        display: block;
+        cursor: pointer;
+
+        &i {
```

Comment:
This CSS syntax is no longer permitted; there's a method in #general for how to have `stylelint` update your PR.

Reply:
used `.zulip-icon` instead of i